.circleci/config.yml

```diff
@@ -54,12 +58,36 @@ jobs:
             make build-darwin VERSION=head
             make publish-head
 
+  test_windows:
+    executor: win/default
+    steps:
+      - checkout:
+          path: ..\go\src\github.com\codeclimate\test-reporter
+      - run:
+          name: Install GVM
+          command: |
+            [Net.ServicePointManager]::SecurityProtocol = "tls12"
+            Invoke-WebRequest -URI https://github.com/andrewkroh/gvm/releases/download/v0.5.0/gvm-windows-amd64.exe -Outfile gvm.exe
+      - run:
+          name: Install golang 1.15
+          command: |
+            .\gvm install 1.15
+            .\gvm --format=powershell 1.15 | Invoke-Expression
+            go version
```

Comment:
Actually, the image that [CircleCI provides](https://circleci.com/developer/machine/image/windows-server-2019) for windows, already include `go`. Here I'm installing it manually because we need that specific version (`1.15`).

Regarding using other orbs, from what I've found, they use the same windows base image. For example [the one you shared](https://circleci.com/developer/orbs/orb/cci-orb/golang#executors-windows):
```
machine:
  image: 'windows-server-2019-vs2019:<< parameters.version >>'
  shell: << parameters.shell >>
parameters:
  shell:
    default: powershell.exe -ExecutionPolicy Bypass
    description: Shell to use for execution command.
    type: string
  version:
    default: stable
    description: The machine image version for windows platform.
    type: string
```
They allow you to choose the image version, but none of the windows image versions include the Golang version we want (they are all way newer). And, for example, the command they include for installing other go versions, doesn't support windows (look at the source [here](https://circleci.com/developer/orbs/orb/cci-orb/golang#commands-install)).
In addition to that, we're using the official CircleCI orb, which I feel it may be more convenient.